tests/test_web_scraper.py

```diff
@@ -50,3 +52,44 @@ def test_return_prepare_by_date_present(self):
     def test_return_prepared_by_date_not_present(self):
         actual = get_prepared_date(self.driver)
         self.assertIsNone(actual, "The returned date is None")
+
```

Comment:
oops need to remove this bit from the PR, didn't realise the shelved code got pushes